pkg/v3/plugin/hooks/add_from_staging.go

```diff
@@ -49,16 +50,40 @@ func (hook *AddFromStagingHook) RunHook(obs *ocr2keepersv3.AutomationObservation
 	}
 
 	results = hook.sorter.orderResults(results, rSrc)
-	if n := len(results); n > limit {
-		results = results[:limit]
-		hook.logger.Printf("skipped %d available results in staging", n-limit)
-	}
-	hook.logger.Printf("adding %d results to observation", len(results))
-	obs.Performable = append(obs.Performable, results...)
+	added, _ := hook.addByPercentageExceeded(obs, limit, results, ocr2keepersv3.ObservationPerformablesLimit)
```

Comment:
not sure how `ocr2keepersv3.ObservationPerformablesLimit` makes sense here as base size? It will be always 100 while I understand base size is the current size in bytes?

Reply:
It doesn't actually, this is a hangover from other testing I was doing, nice catch!!